http_validator.go

```diff
@@ -25,6 +25,12 @@ type UnsupportedError struct {
 	Cause error
 }
 
+var oasLoader *openapi3.Loader
+
+func init() {
+	oasLoader = openapi3.NewLoader()
+}
```

Comment:
How can we implement singleton in golang?
I'm thinking that this process doesn't have to be that thread-safe.

Reply:
NewLoader() may be executed only when openapi3.Loader is needed (e.g. sync.Once).

Reply:
However, the performance is good enough to be better than it is now, so I think it is fine as it is.
(We can improve it another time.)